### PR TITLE
Fix `length-zero-no-unit` false positives on new math functions

### PR DESCRIPTION
**Pull request by romainmenke**
> Which issue, if any, is this issue related to?

Closes https://github.com/stylelint/stylelint/issues/6870

> Is there anything in the PR that needs further explanation?

e.g. "No, it's self-explanatory."

### CODE REVIEW

File: lib/reference/functions.js [hunk 1]
```diff
@@ -14,7 +14,28 @@ const camelCaseFunctions = new Set([
...
+	'atan',
+	'atan2',
+	'calc',
+	'clamp',
+	'cos',
+	'exp',
+	'hypot',
+	'log',
+	'max',
+	'min',
```
Comment (Mouvedia) [inline lib/reference/functions.js:30]: Could you add `minmax` and `sign`?
https://developer.mozilla.org/en-US/docs/Web/CSS/minmax
https://developer.mozilla.org/en-US/docs/Web/CSS/sign

Comment (romainmenke) [reply lib/reference/functions.js:30]: Added `sign`.

`minmax` is not a math function, so I didn't add that in this list.

File: .changeset/calm-rivers-wink.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `length-zero-no-unit` false positive on new math functions
```
Comment (ybiquitous) [inline .changeset/calm-rivers-wink.md:5]:
```suggestion
Fixed: `length-zero-no-unit` false positives on new math functions
```

**Review by ybiquitous — APPROVED** [review]
Thank you! LGTM 👍🏼

File: .changeset/calm-rivers-wink.md (hunk 2 above)
Comment (romainmenke) [reply .changeset/calm-rivers-wink.md]: good catch, thank you 🙇

**Pull request merged**